examples/hacktoberfest-diagram/hacktoberfest-diagram.js

```diff
@@ -0,0 +1,406 @@
+        style = styleSheet.styles[ShapeBpmnElementKind.GATEWAY_EXCLUSIVE];
+        style[mxConstants.STYLE_FILLCOLOR] = blueMedium;
+
+        // USER TASK
+        style = styleSheet.styles[ShapeBpmnElementKind.TASK_USER];
+        style[mxConstants.STYLE_FILLCOLOR] = '#355571';
+
+        // CALL ACTIVITY
+        style = styleSheet.styles[ShapeBpmnElementKind.CALL_ACTIVITY];
+        style[mxConstants.STYLE_FILLCOLOR] = blueSuperLight;
+        style[mxConstants.STYLE_FONTCOLOR] = blueMedium;
+
+        // POOL
+        style = styleSheet.styles[ShapeBpmnElementKind.POOL];
+        style[mxConstants.STYLE_SWIMLANE_FILLCOLOR] = blueDark;
+        style[mxConstants.STYLE_FILLCOLOR] = blueMedium;
+        style[mxConstants.STYLE_FONTSIZE] = 16;
+    }
+
+}
+
+class HacktoberfestDarkThemeIconPainter extends IconPainter {
+    // START TIMER
+    paintClockIcon({ c, ratioFromParent, setIconOrigin, shape, icon }) {
+        c.setStrokeColor(pinkDark);
+        super.paintClockIcon({ c, ratioFromParent, setIconOrigin, shape, icon });
+    };
+
+    // EXCLUSIVE GATEWAY
+    paintXCrossIcon({ c, ratioFromParent, setIconOrigin, shape, icon }) {
+        icon.strokeColor = blueSuperLight;
+        super.paintXCrossIcon({ c, ratioFromParent, setIconOrigin, shape, icon });
+    };
+
+    // USER TASK
+    paintUserIcon({ c, ratioFromParent, setIconOrigin, shape, icon }) {
+        icon.strokeColor = blueSuperLight;
+        super.paintUserIcon({ c, ratioFromParent, setIconOrigin, shape, icon });
+    };
+
+    // CALL ACTIVITY
+    paintExpandIcon({ c, ratioFromParent, setIconOrigin, shape, icon }) {
+        c.setStrokeColor(blueDark);
+        super.paintExpandIcon({ c, ratioFromParent, setIconOrigin, shape, icon });
+    };
+}
+
+IconPainterProvider.set(new HacktoberfestDarkThemeIconPainter());
+
+
+const bpmnVisualizationHacktoberfestDarkTheme = new BpmnVisualizationHacktoberfestDarkTheme('graphHacktoberfestDarkTheme');
+bpmnVisualizationHacktoberfestDarkTheme.graph.getDefaultParent().setStyle(`${mxConstants.STYLE_FILLCOLOR} = ${blueDark}`) ;
+bpmnVisualizationHacktoberfestDarkTheme.load(bpmn);
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Update cells with new project name
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+function updateCellLabel(bpmnVisualization, cellId, value) {
+    const cell = bpmnVisualization.graph.model.getCell(cellId);
+    bpmnVisualization.graph.model.setValue(cell, value);
+}
+
+function updateCellsLabel(cellId, value) {
+    updateCellLabel(bpmnVisualization, cellId, value);
+    updateCellLabel(bpmnVisualizationHacktoberfestLightTheme, cellId, value);
+    updateCellLabel(bpmnVisualizationHacktoberfestDarkTheme, cellId, value);
+}
+
+inputProjectName.oninput = function(event) {
+    let projectName = event.target.value;
+    bpmn = bpmnDiagram(projectName);
```

Comment:
:question: why do we need to update the bpmn content? it is not used anymore after loaded. And the oninput function is registered after bpmn content as been updated, so it cannot have impact on the  loaded bpmn.

Reply:
I forgot to delete this.
In the first step, I tried to reload all the graph with the new BPMN file, but all the renders mixed and it was very painful to save/restore them.